clients/python/sust/api/climate_explorer/client.py

```diff
@@ -0,0 +1,52 @@
+from sust.api.climate_explorer import clientgen
+from sust.api.climate_explorer.clientgen.api import portfolios_api
+
+
+class Client:
+    def __init__(self, api_key, project=None, endpoint=None):
+        self._default_req_kwargs = {}
+        if project:
+            self._default_req_kwargs['project'] = project
+
+        self._openapi_config = clientgen.Configuration(host=endpoint)
+        self._openapi_config.api_key['api_key'] = api_key
+
+        openapi_client = clientgen.ApiClient(self._openapi_config)
+        self._openapi_instance = portfolios_api.PortfoliosApi(openapi_client)
+
+    def portfolios(self):
+        return self._openapi_instance.portfolios_list(**self._default_req_kwargs)
+
+    def portfolio_assets(self, portfolio_name):
+        return PageIterator(self._openapi_instance.portfolios_assets_list, (portfolio_name,), self._default_req_kwargs)
+
+
+class PageIterator:
```

Comment:
I will write unit tests for this before it lands. Note that it is just a bit of code that understands how to repeatedly call a provided function, managing the "rows" and "page" kwargs to navigate the Climate Explorer API pagination mechanism.